src/radius.rs

```diff
@@ -34,31 +34,26 @@ pub struct RadiusData<'a> {
     pub attributes: Option<Vec<RadiusAttribute<'a>>>,
 }
 
-
 impl<'a> RadiusData<'a> {
     pub fn get_code(&self) -> Option<RadiusCode> {
         RadiusCode::from_u8(self.code)
     }
 }
 
-
-pub fn parse_radius_data(i:&[u8]) -> IResult<&[u8],RadiusData> {
-    do_parse!(i,
-        c:    be_u8 >>
-        id:   be_u8 >>
-        len:  be_u16 >>
-        auth: take!(16) >>
-        attr: cond!(len > 20,
-                    flat_map!(take!(len - 20),complete!(many1!(parse_radius_attribute)))
-        ) >>
-        (
-            RadiusData {
-                code: c,
-                identifier: id,
-                length: len,
-                authenticator: auth,
-                attributes: attr,
-            }
-        )
+pub fn parse_radius_data(i: &[u8]) -> IResult<&[u8], RadiusData> {
+    do_parse!(
+        i,
+        c: be_u8 >> id: be_u8 >> len: be_u16 >> auth: take!(16)
```

Comment:
Indentation broken (and quite unreadable) - I don't think cargo-fmt is doing the right thing here. Please keep the previous indentation